packages/swapper/src/swappers/CowSwapper/CowSwapper.test.ts

```diff
@@ -232,7 +232,7 @@ describe('cowApi', () => {
               networkName: 'Ethereum',
               precision: 18,
               color: '#5C6BC0',
-              icon: 'https://assets.coincap.io/assets/icons/256/eth.png',
+              icon: '',
```

Comment:
```suggestion
              icon: 'https://coin-images.coingecko.com/coins/images/279/large/ethereum.png?1696501628',
```

Reply:
Fixed in [d50ae4f](https://github.com/shapeshift/web/pull/7298/commits/d50ae4fbe0b84892069e74dfc28a17a7599c9e16)